src/lib/utils/renderTemplate.ts

```diff
@@ -1,17 +1,27 @@
-import hogan from 'hogan.js';
+import hogan, { HoganOptions, Template } from 'hogan.js';
+import { Templates, HoganHelpers } from '../../types';
+import { BindEventForHits } from './createSendEventForHits';
+
+type TransformedHoganHelpers = {
+  [helper: string]: () => (text: string) => string;
+};
 
 // We add all our template helper methods to the template as lambdas. Note
 // that lambdas in Mustache are supposed to accept a second argument of
 // `render` to get the rendered value, not the literal `{{value}}`. But
 // this is currently broken (see https://github.com/twitter/hogan.js/issues/222).
-function transformHelpersToHogan(helpers = {}, compileOptions, data) {
-  return Object.keys(helpers).reduce(
+function transformHelpersToHogan(
+  helpers: HoganHelpers = {},
+  compileOptions?: HoganOptions,
+  data?: Record<string, any>
+) {
+  return Object.keys(helpers).reduce< TransformedHoganHelpers>(
```

Comment:
There's a leading space on the generic type here @Haroenv.

Reply:
thanks, that's due to the GitHub suggestion, will fix on master